mappings/net/minecraft/test/TestImplementation.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_6302 net/minecraft/test/TestImplementation
```

Comment:
Is `@TestImplmplentation` explicitly referred in decompiled strings? Otherwise, I would prefer a simpler annotation name like `@GameTest` or such.

Reply:
Yes, I agree. `GameTest` seems better for this.